packages/readr/pages/index.tsx

```diff
@@ -168,6 +172,18 @@ export const getServerSideProps: GetServerSideProps<PageProps> = async ({
 
   try {
     {
+      // fetch mass recall display URL
+      const massRecallDisplayJsonData = await fetch(
+        MASS_RECALL_DISPLAY_JSON_URL,
+        {
+          next: { revalidate: 0 },
+        }
+      ).then((res) => res.json())
+
+      isMassRecall2025 =
+        massRecallDisplayJsonData[
```

Comment:
```suggestion
        massRecallDisplayJsonData?.[
```
[nit] 雖然有 try catch 了但和其他 fetch 綁在一起，怕這裡爆 error 整組壞光光，如果加一次判斷比較保險。

Reply:
好！已加